@navikt/aksel-stylelint/src/deprecations.ts

```diff
@@ -60,4 +60,12 @@ export const deprecations: DeprecatedList = [
     classes: ["navds-textarea__wrapper"],
     message: "Removed in v6.0.0",
   },
+  {
+    classes: [
+      "navds-pagination__prev-next-icon",
+      "navds-pagination__next-text",
+      "navds-pagination__prev-text",
+    ],
+    message: "Removed in v7.1.0. These classes had not effect and were removed",
```

Comment:
Typo, and maybe avoid writing "removed" twice.
```suggestion
    message: "These classes had no effect and were removed in v7.1.0",
```
```suggestion
    message: "Removed in v7.1.0. These classes had no effect",
```